content/docs/install/manual_demo.md

```diff
@@ -41,24 +41,24 @@ unzip osm.zip
 .\windows-amd64\osm.exe version
 ```
 
-The `osm` CLI can be compiled from source using [this guide](/docs/install/).
+You can compile the `osm` CLI from source using [this guide](/docs/install/).
 
 
 
 ## Installing OSM on Kubernetes
 
-With the `osm` binary downloaded and unzipped, we are ready to install Open Service Mesh on a Kubernetes cluster:
+After you download and unsip the `osm` binary, you are ready to install Open Service Mesh on a Kubernetes cluster:
 
-The command below shows how to install OSM on your Kubernetes cluster.
-This command enables
+
+The Install command enables:
 [Prometheus](https://github.com/prometheus/prometheus),
 [Grafana](https://github.com/grafana/grafana), and
 [Jaeger](https://github.com/jaegertracing/jaeger) integrations.
 The `OpenServiceMesh.enablePermissiveTrafficPolicy` chart value instructs OSM to ignore any policies and
 let traffic flow freely between the pods. With Permissive Traffic Policy mode enabled, new pods
 will be injected with Envoy, but traffic will flow through the proxy and will not be blocked.
 
-> Note: Permissive Traffic Policy mode is an important feature for brownfield deployments, where it may take some time to craft SMI policies. While operators design the SMI policies, existing services will to continue to operate as they have been before OSM was installed.
+> Note: Permissive Traffic Policy mode is an important feature for brownfield deployments (where it may take some time to craft SMI policies). Operators can design the SMI policies while existing services continue to operate as they have been before you installed OSM.
```

Comment:
On line 65, we have
```
    --set=OpenServiceMesh.enablePermissiveTrafficPolicy=true
```

Does that work as expected for you? AFAICT there is a missing trailing `\`.

Reply:
I do see how subsequent commands do have a trailing \ but I didn't see any errors on that part of the install.